Include/cpython/initconfig.h

```diff
@@ -147,6 +147,10 @@ typedef struct {
        Set to 1 by -X faulthandler and PYTHONFAULTHANDLER. -1 means unset. */
     int faulthandler;
 
+    /* Enable PEG parser?
+       1 by default, set to 0 by -X oldparser and PYTHONOLDPARSER */
+    int use_peg;
```

Comment:
I suggest to rename the option to "_use_peg_parser". If you are not used to parsers, it's hard to guess "peg" meaning. I prefer to add "_" prefix to make it private since it's a temporary option for Python 3.9, will be removed from 3.10.

The new member should be documented at https://docs.python.org/dev/c-api/init_config.html#pyconfig with a ".. versionadded:: 3.9" markup. Again, I suggest to directly deprecate it and announce its removal in 3.10 using ".. deprecated-removed" markup.